docs/reference/msbuild-targets.md

```diff
@@ -251,6 +251,25 @@ When packing a license file, you need to use PackageLicenseFile property to spec
 
 [License file sample](https://github.com/NuGet/Samples/tree/master/PackageLicenseFileExample).
 
+### Packing a file without an extension
+
+In some scenarios, like when packing a license file, you might want to include a file without an extension.
+For historical reasons, NuGet & MSBuild treat paths without an extension as directories
```

Comment:
Woah this is some wizardy. Should we specifically call out what `Visible` is and what it does here? Is there a link to docs we could add?

Reply:
I'll just remove it :D 
I don't want to distract from the point of this change.